src/AndersonAcceleration.f90

```diff
@@ -29,21 +30,11 @@ MODULE AndersonAccelerationModule
 PUBLIC :: AndersonAccelerationType
 
 !> @brief the native Anderson Accelerator Type
-TYPE :: AndersonAccelerationType
-  !> Initialization status
-  LOGICAL(SBK) :: isInit=.FALSE.
-  !> Size of solution vector/matrix
-  INTEGER(SIK) :: n=-1
-  !> Current iteration count
-  INTEGER(SIK) :: s=0
+TYPE,EXTENDS(SolutionAccelerationType) :: AndersonAccelerationType
   !> Iteration depth of Anderson solver
   INTEGER(SIK) :: depth=1
-  !> Starting iteration for Anderson
-  INTEGER(SIK) :: start=1
   !> Value of mixing parameter
   REAL(SRK) :: beta=0.5_SRK
```

Comment:
Seems like the mixing parameter for Anderson (beta) and the under-relaxation parameter (alpha) for the Picard classes could just be put on the base class as beta and combined. They both serve the same function.

Reply:
my basic thought on this was they are called different things in the literature.  I also don't think there's any literature out there that would allow beta > 1, but there'a a lot out there on overrelaxing.  I'm not just thinking about this for coupling, but potentially replacing our current relaxation for currents in cmfd, etc.  Basically anywhere we have a relaxation applied, to use this class... then allow us to more easily experiment with trying Anderson in other areas.

Reply:
Okay. I'm good with leaving them differentiated.